apps/aecore/test/aec_chain_tests.erl

```diff
@@ -605,6 +605,7 @@ forking_test_() ->
      , {"Test if hash is connected to genesis", fun fork_is_connected_to_genesis/0}
      , {"Test if hash is in main chain", fun fork_is_in_main_chain/0}
      , {"Get a transaction from the right fork", fun fork_get_transaction/0}
+     , {"Fork on micro-block", fun fork_on_micro_block/0}
```

Comment:
By commenting code changes in this PR this test fails with `illegal_micro_fork`. Good!